std/algorithm/searching.d

```diff
@@ -3913,72 +3913,81 @@ Do nothing if there is no match.
 Params:
     pred = The predicate that determines whether elements from each respective
         range match. Defaults to equality $(D "a == b").
+*/
+template skipOver(alias pred = "a == b")
```

Comment:
This is the eponymous pattern also used by other methods in this module (e.g. [canFind](https://github.com/dlang/phobos/blob/master/std/algorithm/searching.d#L2495), [all](https://github.com/dlang/phobos/blob/master/std/algorithm/searching.d#L118) or [any](https://github.com/dlang/phobos/blob/master/std/algorithm/searching.d#L165)).